consensus/clique/src/main/java/tech/pegasys/pantheon/consensus/clique/headervalidationrules/VoteValidationRule.java

```diff
@@ -33,7 +33,7 @@
   @Override
   public boolean validate(final BlockHeader header, final BlockHeader parent) {
     final long nonce = header.getNonce();
-    if (!VoteType.fromNonce(nonce).isPresent()) {
+    if (!CliqueVotingBlockInterface.voteToValue.values().contains(nonce)) {
```

Comment:
This feels like it should be a static `CliqueVotingBlockInterface.isValidNonce` method rather than reaching into the classes internals.

Reply:
Done.